pkg/codegen/jenny_go_resources.go

```diff
@@ -57,7 +58,15 @@ func (ag *ResourceGoTypesJenny) Generate(kind kindsys.Kind) (*codejen.File, erro
 	if err != nil {
 		return nil, err
 	}
-	return codejen.NewFile(fmt.Sprintf("pkg/kinds/%s/%s_gen.go", mname, mname), buf.Bytes(), ag), nil
+
+	f := codejen.NewFile(fmt.Sprintf("pkg/kinds/%s/%s_gen.go", mname, mname), buf.Bytes(), ag)
+	content, err := format.Source(f.Data)
```

Comment:
Is there any reason to call `format.Source` over `f.Data` instead of calling it over `buf.Bytes()` and then generating the file?

Looks like the latter would be a little bit more clear.

In any case, I think the end result is fine either way! :)

Reply:
Fair, fair enough! hahahaha